src/components/Quote.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+
+const Quote = () => {
+  const [quote, setQuote] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      const url = 'https://api.api-ninjas.com/v1/quotes?category=love';
+      const headers = { 'X-Api-Key': 'aL4yRrpcK1ykMPWPi+rSeA==TEXP1k8ppWk6Zi9w' };
+      try {
+        const response = await fetch(url, { headers });
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const result = await response.json();
+        setQuote(result[0]);
+        setLoading(false);
+      } catch (error) {
+        setError(error.message);
+        setLoading(false);
+      }
+    };
+    fetchQuote();
+  }, []);
```

Comment:
- [ ] Good job on your implementation. Things are working as well but it's highly considered to pass in dependencies to the dependency array of the useEffect hook. Any variable used in the hook that is not created inside it should be passed as a dependency array. Kindly add them 👍🏽

```javascript
 fetchQuote();
  }, [setQuote, setLoading]);
```

Reply:
@AmaduKamara Thanks for your review